pkg/dataobj/sections/logs/table_build.go

```diff
@@ -22,7 +23,14 @@ func buildTable(buf *tableBuffer, pageSize, pageRowCount int, compressionOpts da
 		messageBuilder   = buf.Message(pageSize, pageRowCount, compressionOpts)
 	)
 
+	var prev Record
 	for i, record := range records {
+		if equalRecords(prev, record) {
+			// Skip equal records
+			continue
```

Comment:
this will add NULLs into table, since `i` is used to indicate the row number and continuing skips over an `i`